Exdi/exdigdbsrvsample/ExdiGdbSrvSample/Microsoft.Debugging.KDNet.ExdiGdbServer.nuspec

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.Debugging.KDNet.kdnet-dev-ext</id>
+        <version>1.0.0</version>
+        <description>This package contains kdnet-dev-ext.dll, which is an extension for kdnet.exe to give it additional functionality for specific devices.</description>

```

Comment:
Is this the correct description for this package? Doesnt seem to have anything to do with kdnet-dev-ext.dll